etc/nscommands

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
This file need Copyright info, similarly there are some other files, where it is missing, please add accordingly.

Reply:
This does not need a license, This is a Nagios way of defining commands. It is basically a configuration file.